avocado/core/test.py

```diff
@@ -999,12 +999,17 @@ class SimpleTest(Test):
     Run an arbitrary command that returns either 0 (PASS) or !=0 (FAIL).
     """
 
+    DATA_SOURCES = ["file", "variant"]
+
     re_avocado_log = re.compile(r'^\d\d:\d\d:\d\d DEBUG\| \[stdout\]'
                                 r' \d\d:\d\d:\d\d WARN \|')
 
     def __init__(self, name, params=None, base_logdir=None, job=None):
         super(SimpleTest, self).__init__(name=name, params=params,
                                          base_logdir=base_logdir, job=job)
+        self._data_sources_mapping = {"file": [lambda: self.datadir],
+                                      "variant": [lambda: self.datadir,
+                                                  lambda: self.name.variant]}
```

Comment:
OK, as you still insist on this let's have it this way. Would you please mention this in documentation as it is inconsistent and might confuse users?

Reply:
I'm not sure I understand what is the other way here...

Reply:
The other way would be to keep the data_sources_mapping the same as for INSTRUMENTED tests. Anyway I don't have a strong opinion, it'd just be easier for me to follow even though it'd always be `test`. (the benefit would be easier contrib scripts to put expected stdout/errs into the expected locations, which will need different treatment for INSTRUMENTED and SIMPLE tests)